contexts/did-core-latest.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "id": "@id",
+    "type": "@type",
+    "dc": "http://purl.org/dc/terms/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "schema": "http://schema.org/",
+    "did": "https://w3c.github.io/did-core-registry/#",
```

Comment:
```suggestion
    "didr": "https://w3c.github.io/did-core-registry/#",
```

`did` is almost certainly going to blow up in our faces... try `didr` instead...